src/utils/fetchPonyfill.ts

```diff
@@ -1,4 +1,5 @@
+import isomorphicFetch from 'isomorphic-fetch';
+
 export default (typeof window !== 'undefined' && window.fetch) || // use buildin fetch in browser if available
   (typeof global !== 'undefined' && global.fetch) || // use buildin fetch in node, react-native and service worker if available
-  // eslint-disable-next-line global-require
-  require('isomorphic-fetch'); // ponyfill fetch in node and browsers that don't have it
+  isomorphicFetch; // ponyfill fetch in node and browsers that don't have it
```

Comment:
does this get bundles even if fetch is available?

Reply:
Yes, however it's relatively small, ~150 lines out of 22896 for the `iife` build.

It felt like a smaller cost than trying to refactor the corresponding ponyfill usage to account for using a dynamic import`.

Reply:
@janek26 what do you think, are we good to merge?